internal/internal_command_state_machine.go

```diff
@@ -1556,8 +1565,19 @@ func (h *commandsHelper) getSignalID(initiatedEventID int64) string {
 	return signalID
 }
 
-func (h *commandsHelper) startTimer(attributes *commandpb.StartTimerCommandAttributes) commandStateMachine {
-	command := h.newTimerCommandStateMachine(attributes)
+func (h *commandsHelper) startTimer(
+	attributes *commandpb.StartTimerCommandAttributes,
+	options TimerOptions,
+	dc converter.DataConverter,
+) commandStateMachine {
+	var summary *commonpb.Payload
+	if options.Summary != "" {
+		var err error
+		if summary, err = dc.ToPayload(options.Summary); err != nil {
+			panic(err)
```

Comment:
This should be rare (it's just serializing string), but there is precedent with other `commandsHelper` calls panicking if they can't convert.